### PR TITLE
add KFP-Tekton contact information here

### PR DESCRIPTION
**Pull request by animeshsingh**
Fixes https://github.com/kubeflow/community/issues/466

### CODE REVIEW

**Comment by animeshsingh** [comment]
@Bobgy please take a look - given main Kubeflow readme is going to point, want to ensure this information is surfaced here

File: wg-pipelines/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The [charter](charter.md) defines the scope and governance of the Pipelines Work
 * James Wu (**[@james-jwu](https://github.com/james-jwu)**), Google
 * Karthik Ramachandran (**[@kramachandran](https://github.com/kramachandran)**), Google
 * Pavel Dournov (**[@paveldournov](https://github.com/paveldournov)**), Google
+* Animesh Singh (**[@animeshsingh](https://github.com/animeshsingh)**), IBM ([KFP-Tekton](https://github.com/kubeflow/kfp-tekton))
```
Comment (Bobgy) [inline wg-pipelines/README.md:22]: Hi @animeshsingh, I think this file is auto generated, you should update according to
https://github.com/kubeflow/community/generator/README.md

Comment (animeshsingh) [reply wg-pipelines/README.md:22]: @Bobgy change is is this PR 
https://github.com/kubeflow/community/pull/477

**Comment by thesuperzapper** [comment]
@animeshsingh I'm going to close this PR as you can't edit the auto-generated file, and I see you have already created https://github.com/kubeflow/community/pull/477 which edits `wgs.yaml`

**Pull request closed without merging**